CONTRIBUTING.md

```diff
@@ -9,7 +9,9 @@ If you are new to using git or the GitHub platform you may find it
 helpful to review [video lecture
 32.8](http://www.math.tamu.edu/~bangerth/videos.676.32.8.html), which
```

Comment:
uh, that's outdated. can you update that as well while there? (tamu->colostate is all you need)